### ISSUE TITLE
Get-started: Rearrange TOC

### ISSUE THREAD

**Issue by hekkim** [getify/You-Dont-Know-JS#1659]
**Please type "I already searched for this issue":**
I already searched for this issue.

**Edition:** (1st or 2nd)
2nd

**Book Title:**
Get-started

**Chapter:**
TOC

**Section Title:**
Iteration

**Problem:**
Currently, `Iteration` is in between `Prototype` and `Asking "Why"?` in TOC like:
```md
* Chapter 3: Digging To The Roots Of JS
    * Closure
    * `this` Keyword
    * Prototypes
    * Iteration
    * Asking "Why?"
```

However, an actual order in chapter 3 is like this.
```md
* Chapter 3: Digging To The Roots Of JS
    * Iteration
    * Closure
    * `this` Keyword
    * Prototypes
    * Asking "Why?"
```

So, I think `Iteration` section in chapter 3 should be listed at first instead of `Closure` since the chapter begins with `Iteration`.

It's truly a minor issue, so if you want I can make a PR.

**Comment by getify** [comment]
Thanks for catching this. Yes a PR is appreciated. :)

**Comment by getify** [comment]
Fixed in #1661
